ansible-server/main.py

```
@@ -5,59 +5,65 @@
-            "event": "end",
-            "data": "Playbook finished"
-        }
+def event_generator():
+    try:
+        thread, runner = ansible_runner.run_async(
+            private_data_dir=".",
+            playbook=playbook,
+            cmdline='--check --diff',
+            quiet=True
+        )
+    except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)